apps/google-analytics-4/frontend/src/components/main-app/AnalyticsApp/AnalyticsApp.tsx

```diff
@@ -53,9 +56,9 @@ const AnalyticsApp = (props: Props) => {
       setLoading(false);
     }
 
-    if (reportSlug && propertyId && !isContentTypeWarning) fetchRunReportData();
+    if (runReportFetchRequirements) fetchRunReportData();
     else setTimeout(() => setLoading(false), 200);
```

Comment:
I know this isn't exactly the code you wrote here, but what is this 200 ms delay doing? seems really strange